docs/developer-guide/syntax-reference.rst

```diff
@@ -1217,22 +1217,40 @@ Scalar functions
-| TOPKDISTINCT           | ``TOPKDISTINCT(col1, k)`` |  Return the distinct Top *K* values for the given column and window |
-+------------------------+---------------------------+---------------------------------------------------------------------+
-
++------------------------+---------------------------+----------------------------------------------------------------------------------+
+| Function               | Example                   | Input Type | Description                                                         |
++========================+===========================+============+=====================================================================+
+| COLLECT_LIST           | ``COLLECT_LIST(col1)``    | Stream,    | Return an array containing all the values of ``col1`` from each     |
+|                        |                           | Table      | input row (for the specified grouping and time window, if any).     |
+|                        |                           |            | Currently only works for simple types (not Map, Array, or Struct).  |
+|                        |                           |            | This version limits the size of the result Array to a maximum of    |
+|                        |                           |            | 1000 entries and any values beyond this limit are silently ignored. |
++------------------------+---------------------------+------------+---------------------------------------------------------------------+
+| COLLECT_SET            | ``COLLECT_SET(col1)``     | Stream     | Return an array containing the distinct values of ``col1`` from     |
+|                        |                           |            | each input row (for the specified grouping and time window, if any).|
+|                        |                           |            | Currently only works for simple types (not Map, Array, or Struct).  |
+|                        |                           |            | This version limits the size of the result Array to a maximum of    |
+|                        |                           |            | 1000 entries and any values beyond this limit are silently ignored. |
++------------------------+---------------------------+------------+---------------------------------------------------------------------+
+| COUNT                  | ``COUNT(col1)``           | Stream,    | Count the number of rows                                            |
+|                        |                           | Table      |                                                                     |
++------------------------+---------------------------+------------+---------------------------------------------------------------------+
+| HISTOGRAM              | ``HISTOGRAM(col1)``       | Stream,    | Returns a map containing the distinct values of ``col1`` mapped to  |
```

Comment:
"Return" to be consistent with other functions.

Reply:
fixed